docs/state.rst

```diff
@@ -244,3 +246,138 @@ For example:
         otherAppOtherAccountRole,
         If(otherAppOtherAccountRole.hasValue(), otherAppOtherAccountRole.value(), Bytes("none"))
     ])
+
+Box Storage
+-----------
+
+Box storage consists of key-value pairs that are stored in an application's local context.
+
+The app account's minimum balance requirement (MBR) is increased with each additional box, and each additional byte in the box'es name and allocated size.
+
+.. warning::
+
+   If one deletes an application with outstanding boxes, the MBR is not recoverable from the deleted app account.
+   It is recommended that *before* app deletion, all box storage be deleted, and funds previously allocated to the MBR be withdrawn.
+
+Box sizes and names cannot be changed after initial allocation, but they can be deleted and re-allocated.
+Boxes are only visible to the application itself; in other words, an application cannot read from or write to another application's boxes on-chain.
+
+Boxes can be manipulated as follows:
```

Comment:
@ahangsu Minor and optional:  I'd expect an immediate listing following a colon (:) rather than a series of subsections.

I don't feel strongly and you're welcomed to resolve as is.  Alternatives include (a) removing the line or (b) rephrasing (_suggestion_ offers one phrasing).

```suggestion
The following sections explain how to work with boxes.
```

Reply:
I would like to rephrase this as you suggested, feels making more sense.